### PR TITLE
Navbar cleanup

### PR DESCRIPTION
**Pull request by clintonhill**
I've completed the general outline and look of the Navbar.

### CODE REVIEW

File: public/javascripts/xp.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*
+ 1: 0 - 1000,
+ 2: 1000 - 3000,
+ 3: 3000 - 6000,
+ 4: 6000 - 10000,
+ etc...
+*/
```
Comment (genagain) [inline public/javascripts/xp.js]: Please either provide more context for this comment, or delete it.

File: public/javascripts/xp.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  const actualLevel = getCurrentLevelFromXP(currentXP);
+  const thisLevelXP = getXPRequiredForLevel(actualLevel);
+  const nextLevelXP = getXPRequiredForLevel(actualLevel + 1);
+
+  const progress = ((currentXP - thisLevelXP) * 100 )/ (nextLevelXP - thisLevelXP);
+
+  currLevel.innerText = actualLevel
+  nextLevel.innerText = getNextLevelFromXP(currentXP);
+  progressBar.style.width = progress + '%';
```
Comment (genagain) [inline public/javascripts/xp.js]: I think it's cleaner to use template literal here `${progress}%`, so please change that.

File: public/stylesheets/navbar.css [hunk 3]
```diff
@@ -1,68 +1,120 @@
...
     background-image: url('../images/first-png.jpg');
 }
 
 .navbar {
     display: flex;
+    height: 4em;
 }
 
 .user_auth {
     /* width: 300px; */
```
Comment (genagain) [inline public/stylesheets/navbar.css]: Please remove this commented out code.

File: public/stylesheets/navbar.css [hunk 4]
```diff
@@ -1,68 +1,120 @@
...
 }
 
 .logo-container {
     display: inline-block;
-    width: 100px;
+    width: 200px;
     height: 50px;
     background-repeat: no-repeat;
-    background-size: contain;
+    background-size:cover;
```
Comment (genagain) [inline public/stylesheets/navbar.css]: Please add a space between the colon and the value.

**Review by genagain — CHANGES_REQUESTED** [review]
Overall looks good to me! Once you've made the changes I requested, I'll approve it.

File: public/javascripts/xp.js (hunk 1 above)
Comment (clintonhill) [reply public/javascripts/xp.js]: I removed it.

File: public/javascripts/xp.js (hunk 2 above)
Comment (clintonhill) [reply public/javascripts/xp.js]: Changed to template literal.

File: public/stylesheets/navbar.css (hunk 3 above)
Comment (clintonhill) [reply public/stylesheets/navbar.css]: Comments all removed.

File: public/stylesheets/navbar.css (hunk 4 above)
Comment (clintonhill) [reply public/stylesheets/navbar.css]: Spaces added.

**Review by genagain — APPROVED** [review]
Looks good to me! Ship it! 🚀

**Pull request merged**